src/createServer.js

```diff
@@ -1,3 +1,46 @@
 // Write code here
 // Also, you can create additional files in the src folder
 // and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+const { errorHandler } = require('./errorHandler');
+
+function createServer() {
+  const server = http.createServer(async (req, res) => {
+    res.setHeader('Content-type', 'application/json');
+
+    const normalizedURL = new URL(req.url, 'http://localhost:5700');
+
+    const originalText = normalizedURL.pathname.slice(1);
+    const targetCase = normalizedURL.searchParams.get('toCase');
+
+    try {
+      if (!originalText) {
+        throw new Error('Text should not be null');
```

Comment:
Instead of throwing a generic Error, consider using a more specific error type or message that can be easily identified in the error handler.